src/api-clients/room-api-client.js

```diff
@@ -51,7 +50,7 @@ class RoomApiClient {
   deleteAllPrivateRoomsForUser({ ownerId }) {
```

Comment:
We could even rename it to `deleteRoomsForUser`

Reply:
Course, forgot the client-side, I had renamed just server-side 😵‍💫 
Thanks, done!